src/main/java/com/whiletrue/tododemo/dto/TaskRequest.java

```diff
@@ -10,17 +10,12 @@ public class TaskRequest {
     private String name;
     private String description;
     private Instant dueDateTime;
-    private String createdBy;
-    private String assignedTo;
     private boolean completed;
 
-    public TaskRequest(String name, String description, Instant dueDateTime, String createdBy, String assignedTo, boolean completed) {
+    public TaskRequest(String name, String description, Instant dueDateTime, boolean completed) {
```

Comment:
Try to integrate Lombok. Here instead of writing a constructor will all of the class properties, you could just use ``@AllArgsConstructor`` annotation